Cargo.toml

```diff
@@ -33,7 +33,7 @@ thiserror = "1.0"
 libc = "0.2.147"
 oci-spec = { version = "0.6.1", features = ["runtime"] }
 sha256 = "1.4.0"
-libcontainer = { version = "0.1", default-features = false }
+libcontainer = { git = "https://github.com/containers/youki", rev = "09e67372a892f22a89eeef62ff429c3cbcac6d41", default-features = false }
```

Comment:
Could you please add a comment saying 

"the tip of the main branch. TODO: once lincontainer releases 0.2, switch to released version"